[2024-08-05T09:06:23.473+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-05T09:06:23.498+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Tokyo manual__2024-08-05T05:57:13.211101+00:00 [queued]>
[2024-08-05T09:06:23.506+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Tokyo manual__2024-08-05T05:57:13.211101+00:00 [queued]>
[2024-08-05T09:06:23.506+0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-05T09:06:23.520+0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_air_pollution_data_Tokyo> on 2024-08-05 05:57:13.211101+00:00
[2024-08-05T09:06:23.524+0300] {standard_task_runner.py:64} INFO - Started process 14722 to run task
[2024-08-05T09:06:23.531+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'air_pollution_data_dag', 'fetch_air_pollution_data_Tokyo', 'manual__2024-08-05T05:57:13.211101+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/air_pollution_data_dag.py', '--cfg-path', '/tmp/tmpnpk_np52']
[2024-08-05T09:06:23.531+0300] {standard_task_runner.py:91} INFO - Job 52: Subtask fetch_air_pollution_data_Tokyo
[2024-08-05T09:06:23.571+0300] {task_command.py:426} INFO - Running <TaskInstance: air_pollution_data_dag.fetch_air_pollution_data_Tokyo manual__2024-08-05T05:57:13.211101+00:00 [running]> on host std21092
[2024-08-05T09:06:23.636+0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='air_pollution_data_dag' AIRFLOW_CTX_TASK_ID='fetch_air_pollution_data_Tokyo' AIRFLOW_CTX_EXECUTION_DATE='2024-08-05T05:57:13.211101+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-05T05:57:13.211101+00:00'
[2024-08-05T09:06:23.637+0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-05T09:06:28.214+0300] {logging_mixin.py:188} INFO - Erreur lors de la récupération des données pour Tokyo
[2024-08-05T09:06:28.214+0300] {python.py:237} INFO - Done. Returned value was: None
[2024-08-05T09:06:28.215+0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-05T09:06:28.222+0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=air_pollution_data_dag, task_id=fetch_air_pollution_data_Tokyo, run_id=manual__2024-08-05T05:57:13.211101+00:00, execution_date=20240805T055713, start_date=20240805T060623, end_date=20240805T060628
[2024-08-05T09:06:28.275+0300] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-05T09:06:28.307+0300] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-05T09:06:28.310+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
